packages/driver/src/cy/commands/actions/scroll.coffee

```diff
@@ -221,15 +221,34 @@ module.exports = (Commands, Cypress, cy, state, config) ->
         $utils.throwErrByPath("scrollTo.invalid_target", {args: { x, y }})
 
       if options.log
-        deltaOptions = $utils.filterOutOptions(options, {duration: 0, easing: 'swing'})
+        deltaOptions = $utils.stringify(
+          $utils.filterOutOptions(options, {duration: 0, easing: 'swing'})
+        )
+
+        messageArgs = []
+        if !position
+          messageArgs.push(x)
+          messageArgs.push(y)
+        else
+          messageArgs.push(position)
+        if deltaOptions
+          messageArgs.push(deltaOptions)
 
         log = {
-          message: deltaOptions
+          message: messageArgs.join(', '),
```

Comment:
For the sake of consistency, could you change the single quotes here and elsewhere in the PR to double quotes?